src/fidasim.f90

```diff
@@ -2305,7 +2324,7 @@ subroutine make_beam_grid
 
     beam_grid%drmin  = minval(beam_grid%dr)
     beam_grid%dv     = beam_grid%dr(1)*beam_grid%dr(2)*beam_grid%dr(3)
-    beam_grid%ntrack = beam_grid%nx+beam_grid%ny+beam_grid%nz
+    beam_grid%ntrack = (beam_grid%nx+beam_grid%ny+beam_grid%nz)*50
```

Comment:
Make `50` a variable defined somewhere at the beginning of the subroutine in case a future developer needs to change the value later.

Reply:
@alvin-garcia I'll change it to be the `max_cell_splits` variable since it corresponds to an upper limit in the track subroutine
https://github.com/D3DEnergetic/FIDASIM/blob/5a813b162653d3dab2f7dce071572a6ee4a97d8a/src/fidasim.f90#L7211-L7215